tracing-subscriber/Cargo.toml

```diff
@@ -49,9 +49,15 @@ serde_json = { version = "1.0", optional = true }
 serde = { version = "1.0", optional = true }
 tracing-serde = { path = "../tracing-serde", optional = true }
 
+# a hack, trying out a formatter
+humantime = "1.2"
```

Comment:
this dependency is no longer used (all the formatters in this PR use `chrono` as they did previously). can we remove it?

Reply:
removed.